Application/Published/Upgrade-AdminApp/Deploy-EdFiOdsAdminApp.ps1

```diff
@@ -0,0 +1,220 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+#Requires -Version 3.0
+
+<#
+.SYNOPSIS
+    Deploys Admin App appliacation to an existing Azure resource group
+.DESCRIPTION
+	Deploys Admin App appliacation to an existing Azure resource group
+.PARAMETER ResourceGroupName
+	Existing resource group name
+.PARAMETER ResourceGroupLocation
+    Existing resource group location
+.PARAMETER AdminAppName
+    A friendly name to help identify AdminApp within Azure.  Must be no more than 64 characters long
+.PARAMETER AppInsightLocation
+	Existing app insight location, mostly same as resouce group location
+.PARAMETER ProductionApiUrl
+	Existing ODS/ API Url that Admin App can connect
+.PARAMETER EncryptionKey
+	Base64-encoded 256 bit key appropriate for use with AES encryption
```

Comment:
Should there be a comment that this is optional, and a key will be created if one is not provided?